### DISCUSSION TITLE
Ability to customize color labels

### DISCUSSION THREAD

**Discussion by OpenFoam-User** [windingwind/zotero-actions-tags#210] [Q&A] [answered]
Hello,

Thank you for this great plugin, it is very useful.

I am asking if it possible to change color labels when highlighting text to make them more meaningful. I mean instead of using 'Yellow', 'Red', 'Green', etc, I want them changed to 'Important', 'Very important', 'Reference', etc.

Something like this:

![image](https://github.com/windingwind/zotero-actions-tags/assets/67183566/2ac75bda-d05c-495b-bb03-1714f5b825b5)

Best regards

**Comment by windingwind** [comment, marked as answer]
#211
